examples/grafana/detail.json

```diff
@@ -529,7 +529,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "sum(rate(pyrra_requests_total{slo=\"$slo\"}[$__rate_interval]))",
+          "expr": "sum(rate(pyrra_requests:rate5m{slo=\"$slo\"}[$__rate_interval]))",
```

Comment:
This doesn't make that much sense anymore. Given we already have the `:rate5m` we don't need to do anything except `pyrra_requests:rate5m{slo=\"$slo\"}`. 
I guess we'll lose the flexibility to use `rate[$__rate_interval]` but being correct is more important.

Reply:
We could have multiple rate interval pre-computed i.e. rate5m, rate1h, rate1d, how about that?